src/main/java/com/google/cloud/genomics/dataflow/pipelines/IdentityByState.java

```diff
@@ -57,11 +61,24 @@ public static void main(String[] args) throws IOException, GeneralSecurityExcept
 
     Pipeline p = Pipeline.create(options);
     DataflowWorkarounds.registerGenomicsCoders(p);
-    DataflowWorkarounds
-        .getPCollection(requests, p, options.getNumWorkers())
-        .apply(
-            ParDo.named(VariantReader.class.getSimpleName()).of(
-                new VariantReader(auth, VARIANT_FIELDS)))
+    PCollection<SearchVariantsRequest> input =
+        DataflowWorkarounds.getPCollection(requests, p, options.getNumWorkers());
+
+    PCollection<Variant> variants;
```

Comment:
Looks like the perfect use case for the ternary operator:

    PCollection<Variant> variants = options.getIsGvcf()
        // Special handling is needed for data in gVCF format since IBS must take into account
        // reference-matches in addition to the variants (unlike
        // other analyses such as PCA).
        ? JoinGvcfVariants.joinGvcfVariantsTransform(input, auth, JoinGvcfVariants.GVCF_VARIANT_FIELDS)
        : input.apply(ParDo.named(VariantReader.class.getSimpleName()).of(new VariantReader(auth, VARIANT_FIELDS)));